packages/flutter_tools/lib/src/base/analyze_size.dart

```diff
@@ -0,0 +1,336 @@
+import 'dart:convert';
+
+import 'package:flutter_tools/src/base/process.dart';
+import 'package:flutter_tools/src/base/terminal.dart';
+import 'package:intl/intl.dart';
+import 'package:meta/meta.dart';
+import 'package:vm_snapshot_analysis/treemap.dart';
+
+import '../base/file_system.dart';
+import '../globals.dart' as globals;
+
+// This file can be in devtools_shared
+class SizeAnalyzer {
+  static const String kAotSizeJson = 'aot-size.json';
+
+  static String getAotSizeAnalysisJsonFile(String aotOutputPath) {
+    return globals.fs.path.join(aotOutputPath, kAotSizeJson);
+  }
+
+  static String getAotSizeAnalysisExtraGenSnapshotOption(String aotOutputPath) {
```

Comment:
avoid method names that start with the word `get`.
https://dart.dev/guides/language/effective-dart/design#avoid-starting-a-method-name-with-get